### ISSUE TITLE
manage.py doesnt load .env

### ISSUE THREAD

**Issue by Mwoua** [vabene1111/recipes#152]
### Version
0.10.1

### Issue
`python3.8 manage.py migrate` doesnt use .env file

### `.env`
Please include your `.env` config file (**make sure to remove/replace all secrets**)
```
DB_ENGINE=django.db.backends.postgresql_psycopg2
POSTGRES_HOST=localhost
POSTGRES_PORT=5432
POSTGRES_USER=djangouser
POSTGRES_PASSWORD=password
POSTGRES_DB=djangodb
```

and `python3.8 manage.py migrate` was still creating db.sqlite3 file

So I updated settings.py file

```
DATABASES = {
    'default': {
        'ENGINE': os.getenv('DB_ENGINE'),
        'HOST': os.getenv('POSTGRES_HOST'),
        'PORT': os.getenv('POSTGRES_PORT'),
        'USER': os.getenv('POSTGRES_USER'),
        'PASSWORD': os.getenv('POSTGRES_PASSWORD'),
        'NAME': os.getenv('POSTGRES_DB')',
    }
}
```
to be sure it use the .env and not the default value. Resulting in an error about DATABASES  struct. So I added `load_dotenv()` at the beginning of the file. And now the database structure is fine

**TLDR**:
Is there a way for `python manage.py migrate` to use the .env file?

(so I can properly write the instructions without docker)

**Comment by vabene1111** [comment]
`python` is run on the shell and thus only has the variables available that are exported there. 

You can use this command to make all variables available to your system `export $(cat .env | xargs)`

**Comment by Mwoua** [comment]
I always forget about xargs :)

I think I have everything to make a complete setup without docker. I'll try to do it later this week.

Should I update the Readme.md in my pull request?

**Comment by vabene1111** [comment]
i think it would be better to create a seperate folder in the /docs/ directory and point to it in the readme or what do you think ?
